lineage/figures/figureCommon.py

```diff
@@ -20,12 +20,12 @@
 
 # bern, gamma_a, gamma_scale
 state0 = StateDistribution(0.99, 7, 7)
-state1 = StateDistribution(0.75, 7, 1)
+state1 = StateDistribution(0.88, 7, 1)
 E = [state0, state1]
 
 # bern, exp_lambda
-state10 = expStateDistribution(0.99, 49)
-state11 = expStateDistribution(0.75, 7)
+state10 = expStateDistribution(1, 49)
```

Comment:
Do you mean for both states to be 1?

Reply:
I just changed them to the original parameters.